src/types/transaction_request.rs

```diff
@@ -97,6 +97,28 @@ mod tests {
         );
     }
 
+    #[test]
+    fn should_deserialize_call_request() {
+        let call_request = CallRequest {
```

Comment:
same here - let's just hardcode a string (or two) and check we deserialize it correctly.